pkg/reduce/options.go

```diff
@@ -11,6 +11,8 @@ type Options struct {
 	readBatchSize int64
 	// windowOpts Options for window
 	windowOpts *window.Options
+	// vertexType indicates the type of the vertex
+	vertexType dfv1.VertexType
```

Comment:
Is this needed? The code for reduce is only supposed to be used for reduce vertex.

Reply:
yeah, I don't think we need that. removed it.